apps/src/block_utils.js

```diff
@@ -1073,13 +1073,14 @@ exports.createJsWrapperBlockCreator = function(
       }
 
       if (expression) {
+        let newExpression = expression.replace('VALUE', values[0]);
```

Comment:
What is newExpression here? Maybe add a comment for what's happening here. Or rename the variable to make it a bit more clear.